SwiftWisdom/Core/StandardLibrary/Numbers/Mathable.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  Mathable.swift
+//  SwiftWisdom
+//
+//  Created by Paul Rolfe on 2/12/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+public protocol Mathable: Comparable {
+    func + (lhs: Self, rhs: Self) -> Self
+    func / (lhs: Self, rhs: Self) -> Self
+    func * (lhs: Self, rhs: Self) -> Self
+    func - (lhs: Self, rhs: Self) -> Self
+    init(_ int: IntMax)
+}
+
+extension Mathable {
+    public static var zero: Self {
+        return self.init(0.toIntMax())
```

Comment:
Because `IntMax` is the type for the Mathable initializer. I'm pretty sure we use that because all number types can initialize from `IntMax` (aka Int64) -- but @LoganWright actually came up with using that initializer for Mathables.